avaframe/in3Utils/cfgUtils.py

```diff
@@ -159,3 +160,38 @@ def compareConfig(iniFile, modName, compare):
         logUtils.writeCfg2Log(cfg, modName)
 
     return cfg
+
+
+def writeCfgFile(avaDir, module, cfg, suffix=''):
+    """ Save configuration used to text file in Outputs """
+
+    # write configuration to ini file
+    modPath = os.path.dirname(module.__file__)
+    # get filename of module
+    name = os.path.basename(module.__file__)
+    modName = name.split('.')[0]
+    # set outputs
+    outDir = os.path.join(avaDir, 'Outputs')
+
+    # write to file
+    with open(os.path.join(outDir, '%s%s_settings.ini' % (modName, suffix)), 'w') as conf:
+        cfg.write(conf)
+
+
+
+def readCfgFile(avaDir, module, suffix=''):
```

Comment:
This function is not really needed as far as I can see, as the function: `cfgUtils.getModuleConfig` 
allows for a fileOverride in which a specified file can be used.

Reply:
I would suggest to keep it as a separate function as the writeCfgFile creates one that is called moduleName_settings.ini, and the read by default reads the moduleName_settings.ini so it appends the correct ending and I think in this way it is less likely to mix it with reading the standard/default moduleNameCfg.ini file..